server/controllers/courses/addCourse.js

```diff
@@ -0,0 +1,22 @@
+const insertCourse = require('../../database/queries/addCourse');
+
+exports.addCourse = (req, res) => {
+  insertCourse({ ...req.body })
+    .then((course) => {
+      if (!course.rowCount) {
+        res.send({
+          error: 'Course not added database',
+          data: null,
+        });
+      } else {
+        res.send({
+          error: null,
+          data: course.rows,
```

Comment:
@AbdallahAmmar96 It might be convenient for the client if you returned `course.rows[0]` since there should always only be 1 row.